optimum/intel/openvino/modeling_visual_language.py

```diff
@@ -2300,33 +2601,26 @@ def preprocess_inputs(
         processor: Optional[AutoImageProcessor] = None,
         tokenizer: Optional[PreTrainedTokenizer] = None,
         config: Optional[PretrainedConfig] = None,
+        video: Optional["VideoInput"] = None,
```

Comment:
why not `videos` to match what the processor expects ?

Reply:
because we already have image instead of images. The main use case of this function is prepare input for single element in batch and plural form will indicate that multiple inputs concatenated in batch is also allowed